app/views/projects/_settings_sidebar.html.erb

```diff
@@ -0,0 +1,24 @@
+<li class="sidebar-item sidebar-brand">
+  PROJECT SETTINGS
+</li>
+
+<% if policy(@project).manage? %>
```

Comment:
You shouldn't use an instance variable into a partial. What do you think to pass this instance variable as a local variable into the partial? 
https://github.com/Codeminer42/cm42-central/pull/91/files#diff-7026f1345bff16328f093505ca9b3db0R11
```
render partial: 'projects/settings_sidebar',
           layout: 'projects/sidebar_wrapper',
           project: @project
```